_posts/2022-10-10-1.16-announcement.md

```diff
@@ -0,0 +1,401 @@
+
+# Stream Processing
+
+In 1.16, we have made improvements in Checkpoints, SQL, Connector and other fields, 
+so that stream computing can continue to lead.
+
+## Generalized incremental checkpoint
+
+Changelog state backend aims at making checkpoint intervals shorter and more predictable, 
+this release is prod-ready and is dedicated to adapting changelog state backend to 
+the existing state backends and improving the usability of changelog state backend:
+ - Support state migration
+ - Support local recovery
+ - Introduce file cache to optimize restoring
+ - Support switch based on checkpoint
+ - Improve the monitoring experience of changelog state backend
+   - expose changelog's metrics
+   - expose changelog's configuration to webUI
+
+Table 1: The comparison between Changelog Enabled / Changelog Disabled on value state 
+(see [this blog](https://flink.apache.org/2022/05/30/changelog-state-backend.html) for more details)
+
+| Percentile  | End to End Duration | Checkpointed Data Size<sup>*<sup> | Full Checkpoint Data Size<sup>*<sup> |
+|-------------|---------------------|-----------------------------------|--------------------------------------|
+| 50%         | 311ms / 5s          | 14.8MB / 3.05GB                   | 24.2GB / 18.5GB                      |
+| 90%         | 664ms / 6s          | 23.5MB / 4.52GB                   | 25.2GB / 19.3GB                      |
+| 99%         | 1s / 7s             | 36.6MB / 5.19GB                   | 25.6GB / 19.6GB                      |
+| 99.9%       | 1s / 10s            | 52.8MB / 6.49GB                   | 25.7GB / 19.8GB                      |
+
+## RocksDB rescaling improvement & rescaling benchmark
+
+Rescaling is a frequent operation for cloud services built on Apache Flink, this release leverages 
+[deleteRange](https://rocksdb.org/blog/2018/11/21/delete-range.html) to optimize the rescaling of 
+Incremental RocksDB state backend. deleteRange is used to avoid massive scan-and-delete operations, 
+for upscaling with a large number of states that need to be deleted, 
+the speed of restoring can be almost [increased by 2~10 times](https://github.com/apache/flink/pull/19033#issuecomment-1072267658).
+
+<center>
+<img src="{{ site.baseurl }}/img/rocksdb_rescaling_benchmark.png" style="width:90%;margin:15px">
+</center>
+
+## Improve monitoring experience and usability of state backend
+
+This release also improves the monitoring experience and usability of state backend.
+Previously, RocksDB's log was located in its own DB folder, which made debugging RocksDB not so easy. 
+This release lets RocksDB's log stay in Flink's log directory by default. RocksDB statistics-based 
+metrics are introduced, statistics-based metrics held at the database level, e.g. total block cache hit 
+count within the DB.
+
+## Support overdraft buffer
+
+A new concept of [overdraft network buffer](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/memory/network_mem_tuning/#overdraft-buffers) 
```

Comment:
```suggestion
A new concept of [overdraft network buffer](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/memory/network_mem_tuning/#overdraft-buffers) 
```
Some doc links to master, and some to 1.16 release. Should we standardise to 1.16 docs?

Reply:
good catch

Reply:
good catch